src/config/app.js

```diff
@@ -26,21 +22,17 @@ app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
 // serve API documentation generated via apidoc at /
-app.use(express.static(path.join(__dirname, "../public")));
+app.use(express.static(path.join(__dirname, '../../public')));
 
 // middleware for verifying token for protected routes
-const unprotectedRoutes = ['/login', '/signup', '/'];
+const unprotectedRoutes = ['/api/login', '/api/signup', '/'];
 app.use(jwt({
```

Comment:
Wondering if login and signup should also be prefixed with `api/`. Thoughts, @mnicole?

Reply:
i think all the routes being at /api is good since they're part of the api too, and in case someday we would want to have a web version like slack does... then it would make sense to have emote.ml/signup be the actual sign up page

Reply:
Thanks! That's what I thought you I remembered you saying earlier about making this change, but just wanted to make sure.